src/ln/channel.rs

```diff
@@ -1863,6 +1872,11 @@ impl Channel {
 					}
 					if err.is_some() {
 						self.holding_cell_htlc_updates.push(htlc_update);
+						if let Some(ChannelError::Ignore(_)) = err {
+							// If we failed to add the HTLC, but got an Ignore error, we should
+							// still send the new commitment_signed, so reset the err to None.
```

Comment:
Maybe log_error'it ? Does your assumption is that strong ? Because if remote get disconnected and backward node keep sending us HTLCs we're gonna to throttle them without sending failure back ?

Reply:
Good idea.

Reply:
You can even match pattern for AddHTLC to log payment_hash/amount, it's should be the only kind of HTLCUpdateAwaitingACK we can have here